src/movies.js

```diff
@@ -1,16 +1,95 @@
 // Iteration 1: All directors? - Get the array of all directors.
+
+function getAllDirectors (movies){
+    let directors = movies.map(function(movie){
+        return movies.directors;
```

Comment:
the return inside the map is having two issues.
- The property of the object is called director, not directors.
- It should be movie, not movies. Movies is the main function parameter and movie is the map parameter. To avoid this kind of issues you can avoid similar naming of your variables. So instead of movies do moviesArray and instead of movie do eachMovie.